### PR TITLE
Converted remaining UI modules to TypeScript

### PR DESCRIPTION
**Pull request by dtdesign**
n/t

### CODE REVIEW

File: wcfsetup/install/files/ts/WoltLabSuite/Core/Bootstrap.js [hunk 1]
```diff
@@ -69,9 +69,7 @@ define(
         DateTimeRelative.setup();
         DatePicker.init();
         UiSimpleDropdown.setup();
-        UiMobile.setup({
-          enableMobileMenu: options.enableMobileMenu
-        });
+        UiMobile.setup(options.enableMobileMenu);
```
Comment (TimWolla) [inline wcfsetup/install/files/ts/WoltLabSuite/Core/Bootstrap.js:72]: Isn't this a BC break?

File: wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Mobile.js [hunk 2]
```diff
@@ -1,377 +1,390 @@
 /**
  * Modifies the interface to provide a better usability for mobile devices.
  *
- * @author	Alexander Ebert
- * @copyright	2001-2019 WoltLab GmbH
- * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
- * @module	WoltLabSuite/Core/Ui/Mobile
+ * @author  Alexander Ebert
+ * @copyright  2001-2019 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Mobile.js:6]: The diff for this file is not really reviewable.

**Review by TimWolla — APPROVED** [review]
Overall this LGTM.

File: wcfsetup/install/files/ts/WoltLabSuite/Core/Bootstrap.js (hunk 1 above)
Comment (dtdesign) [reply wcfsetup/install/files/ts/WoltLabSuite/Core/Bootstrap.js:72]: Yes, but this is an internal module not referenced anywhere else.

**Pull request merged**